### ISSUE TITLE
Windows Defender detects Trojan

### ISSUE THREAD

**Issue by dodekaphilist** [Nama/SuckControl#1]
I was not able to download the executable due to Windows Defender blocking it. It detected the file as a [trojan](https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?name=Trojan%3aWin32%2fWacatac.B!ml).
I don't know how to resolve this issue, but you should probably also think about signing the application to resolve the Windows SmartScreen warning.

**Comment by Nama** [comment]
Yes, that's known and already stated in the readme.
https://stackoverflow.com/questions/54733909/windows-defender-alert-users-from-my-pyinstaller-exe

Signing requires a certificate, which is not free.

I uploaded the same exe as zip and 7z files, try to download them.
https://github.com/Nama/SuckControl/releases/tag/1.1.1

I was also able to download and execute the exe file in Firefox (and Edge, but that required more steps).
On the SmartScreen warning, you have to click on "More information", then the "Run anyway" button appears.

On Edge, you need to click on the 3 dots and choose to keep the file. That also requires you to click on "More Information" before you need to click on "keep" again.

I should add these info to the readme, thanks!

**Comment by dodekaphilist** [comment]
Thanks for your answer.
For me, it doesn't matter how I download it. Both in Edge and Firefox as .zip or .exe, the Windows Defender deletes it. While the file certainly can be restored manually, it's kind of a mess.
As you already said, it might be beneficial to more explicitly clarify that Microsoft has some issues and how to deal with it.

**Comment by Nama** [comment]
> Windows Defender deletes

Didn't for me, but I had this reported some time ago, too. I even scanned SuckControl.exe manually and Windows Defender didn't find anything...
Just uploaded for false detection on MS, nothing detected:
![grafik](https://user-images.githubusercontent.com/2297270/135628050-84091d5c-bfdd-4810-913e-53c2edb7dfce.png)

If I assume that you have the newest Defender definition, then this is weird and I have no idea how to fix this.

**Comment by dodekaphilist** [comment]
I just checked, there are no updates available neither on Windows Update nor in Windows Defender, so I guess I'm up to date.
But it's kind of odd... When I downloaded zip it was blocked, however after I repeated it, everything was fine and a file check with the Defender also showed no issues. It might be resolved by itself but I guess keeping track of this problem would be good. Closing this issue for now.
